### ISSUE TITLE
[Feature Request] After searching, pop matching tabs out in a new window

### ISSUE THREAD

**Issue by joeytwiddle** [josh-berry/tab-stash#346]
TLDR: I realised this feature, whilst useful, might not be needed, since it is provided by other extensions.

### Problem Statement

This is not really specific to TabStash, but since I am using TabStash for my workflow, it would be useful to have. Example:

I have 200 tabs in my main window.

I hit new tab and start to google for something, and open 5 more tabs on the topic.

Then I realise I wish I had started this work in a new window!

Another situation: I might realise I have too many tabs open, and decide I want to move all YouTube tabs to a different window.

### Preferred Solution(s)

In "Tab Manager Plus for Chrome" it lets me do the following:

- Open the extension, which shows a list/grid of tabs
- Use a search to highlight certain windows,
- and/or Shift-click on the displayed tabs to highlight/unhighlight them
- I can press the "Move Tabs to New WIndow" button

I think the highlighting/unhighlighting of certain tabs might be too much extra work for TabStash, for this feature alone.

But a simple button to pop out all search results from the current window, would satisfy me 80% of the time.

### Alternative Solution(s)

Currently what I do is: Stash the tabs I want to pop out one by one. Then create a new window. Then unstash them!

### Additional Context

Oh, I have just discovered that this extension exists for Firefox already!

https://addons.mozilla.org/en-US/firefox/addon/tab-manager-plus-for-firefox/

So perhaps TabStash doesn't really need this feature.

Here is a visual example of using the feature:

![joey's desktop on dumpling at 3:58:51 pm on Thursday 16 March 2023](https://user-images.githubusercontent.com/911799/225552436-06e9a024-7906-420f-af25-8013b520a311.png)

<hr>
<em>Readers: If you are also interested in seeing this feature be developed, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the feature request, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
Thanks for the feature request! This is probably something I'd cover as part of #314. Just to keep things organized, I'll go ahead and mark this as a duplicate of that one, and I'll make a note there that the ability to create a new window from selected/specific tabs is important.
